normandy/recipes/apps.py

```diff
@@ -11,4 +12,5 @@ class RecipesApp(AppConfig):
 
     def ready(self):
         checks.register()
+        RemoteSettings().check_config()
```

Comment:
Where should I add a test for this?

Reply:
I don't think we need to test the `ready` method. I think the tests you have for `check_config` below are good enough.